ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-master.yaml

```diff
@@ -89,6 +61,8 @@ operator:
     with: stable:metering-presto
   - pullspec: quay.io/openshift/origin-metering-reporting-operator:4.8
     with: pipeline:metering-reporting-operator
+  - pullspec: quay.io/openshift/origin-oauth-proxy:4.8
+    with: pipeline:oauth_proxy
```

Comment:
@bparees Do you know if this is required to use `pipeline`, or should I just use `stable` as the `tag_specification` specifies the 4.8 imagestream tag?

Reply:
The `tag_specification` for the master branch configuration:

https://github.com/openshift/release/blob/master/ci-operator/config/kube-reporting/metering-operator/kube-reporting-metering-operator-master.yaml#L100-L102

Reply:
i would expect you can use stable, and you should not need to define it in base_images above, since it's already part of the tag_specification stable is based on.

Reply:
Hmm weird I would expect it to be in the stable tag too after re-reading the CI documentation but the ci-index failures tell a different story. I had already checked that there's an oauth-proxy image tagged in the 4.8 image stream, but not a big deal in either case.

Reply:
it exists in the ocp/4.8:oauth-proxy so i'd expect it to be tagged into your stable imagestream as stable:oauth-proxy.  Very weird and might be worth checking in with the DPTP folks about.